pkg/controller/node/node_controller.go

```diff
@@ -963,6 +974,10 @@ func (ctrl *Controller) updateCandidateMachines(pool *mcfgv1.MachineConfigPool,
 		if err := ctrl.setDesiredMachineConfigAnnotation(node.Name, targetConfig); err != nil {
 			return goerrs.Wrapf(err, "setting desired config for node %s", node.Name)
 		}
+		ctrl.logPool(pool, "Apply UpdateInProgress taint %s target to %s", node.Name, targetConfig)
+		if err := ctrl.setUpdateInProgressTaint(node.Name); err != nil {
+			return goerrs.Wrapf(err, "failed applying updateinprogress taint for node %s", node.Name)
```

Comment:
possibly pull the key out of `NodeUpdateTaint.Key` here and a few lines above, so you don't have to worry about matching spelling/casing/typos.

Reply:
fixed as of  652de25.